modules/server/src/main/scala/navigate/server/NavigateEngine.scala

```diff
@@ -349,17 +350,9 @@ object NavigateEngine {
 
     override def getGuideState: F[GuideState] = systems.tcsCommon.getGuideState
 
-    override def getGuidersQuality: F[GuidersQualityValues] =
-      for {
-        p1Cnts <- (1000 + scala.util.Random.between(-100, 100)).pure[F]
-        p2Cnts <- (1000 + scala.util.Random.between(-100, 100)).pure[F]
-        oiCnts <- (1000 + scala.util.Random.between(-100, 100)).pure[F]
-      } yield GuidersQualityValues(
-        pwfs1 = GuiderQuality(p1Cnts, false),
-        pwfs2 = GuiderQuality(p2Cnts, false),
-        oiwfs = GuiderQuality(oiCnts, false)
-      )
+    override def getGuidersQuality: F[GuidersQualityValues] = systems.tcsCommon.getGuideQuality
```

Comment:
I connected it to the real implementation, and moved the simulation to `TcsBaseControllerSim`